src/sage/arith/misc.py

```diff
@@ -1970,12 +1970,22 @@ def xgcd(a, b):
 
     INPUT:
 
+    One of the following:
+
     -  ``a, b`` -- integers or more generally, element of a ring for which the
        xgcd make sense (e.g. a field or univariate polynomials).
 
+    -  ``a`` -- list or tuple of at least two integers or more generally, element
+       of a ring which the xgcd make sense.
+
     OUTPUT:
 
-    -  ``g, s, t`` -- such that `g = s\cdot a + t\cdot b`
+    One of the following:
+
+    -  ``g, s, t`` -- when two inputs ``a, b`` are given. They satisfy `g = s\cdot a + t\cdot b`.
+
+    -  ``r`` -- when only ``a`` is given (and ``b = None``). The tuple ``r`` has length one longer
```

Comment:
```suggestion
    -  ``r`` -- a tuple, when only ``a`` is given (and ``b = None``). It has length one longer
```
Might have to line wrap this.